packages/table/src/headers/columnHeaderCell.tsx

```diff
@@ -131,18 +136,19 @@ export class ColumnHeaderCell extends React.Component<IColumnHeaderCellProps, IC
     };
 
     public render() {
-        const { className, isActive, isColumnSelected, loading, resizeHandle, style } = this.props;
+        const { loading } = this.props;
```

Comment:
I don't think that's true. I think it's pretty common to destructure all the props you're going to use even if you use them only once.

Also, it's not like this PR is small, so adding a few extra lines in nbd. Also maybe that'll help you avoid lowering the test coverage threshold.